### PR TITLE
[CM-989] Fixed Form Cropping issue for iPhone 6

### PR DESCRIPTION
**Pull request by Sathyan-Elangovan**
## Summary
- Fixed Form Cell height issue for iPhone 6
- Fixed Form Submit Button Width going beyond screen width issue

### CODE REVIEW

File: Sources/Views/ALKFormCell.swift [hunk 1]
```diff
@@ -147,8 +147,8 @@ class ALKFormCell: ALKChatBaseCell<ALKMessageViewModel>, UITextFieldDelegate, UI
 
     private func setUpTableView() {
         itemListView.backgroundColor = .white
-        itemListView.estimatedRowHeight = 40
-        itemListView.estimatedSectionHeaderHeight = 40
+        itemListView.estimatedRowHeight = 50
```
Comment (mukeshthawani) [inline Sources/Views/ALKFormCell.swift:150]: What's the impact of this on iPhone 11 or 11 max?
Add before and after images of those as well. Also, check form buttons that contain text of more than one line

Comment (Sathyan-Elangovan) [reply Sources/Views/ALKFormCell.swift:150]: Checked in iphone11 &  iPhone 11 Pro Max.adding Screenshots here. Also checked multiline button as well
iPhone 11 Pro Max  Before

iPhone 11 Pro max - After

iPhone 11 (15 2) Before
iphone 11 (15 2)- After
.

File: Sources/Views/ALKFriendFormCell.swift [hunk 2]
```diff
@@ -174,3 +175,21 @@ class ALKFriendFormCell: ALKFormCell {
...
 }
+extension UIView {
+    
+    ///// Adds constraints to this `UIView` instances `superview` object to make sure this always has the same size as the superview.     /// Please note that this has no effect if its `superview` is `nil` – add this `UIView` instance as a subview before calling this.
+     func bindFrameToSuperviewBounds() {
+         guard let superview = self.superview
+         else {
+             print("Error! `superview` was nil – call `addSubview(view: UIView)` before calling `bindFrameToSuperviewBounds()` to fix this.")
+             return
+
```
Comment (mukeshthawani) [inline Sources/Views/ALKFriendFormCell.swift:186]: Remove empty line.

File: Sources/Views/ALKFriendFormCell.swift [hunk 3]
```diff
@@ -174,3 +175,21 @@ class ALKFriendFormCell: ALKFormCell {
         nameLabel.isHidden = KMCellConfiguration.hideSenderName
     }
 }
+extension UIView {
+    
+    ///// Adds constraints to this `UIView` instances `superview` object to make sure this always has the same size as the superview.     /// Please note that this has no effect if its `superview` is `nil` – add this `UIView` instance as a subview before calling this.
```
[the remark this answers is not in the document]
Comment (Sathyan-Elangovan) [reply Sources/Views/ALKFriendFormCell.swift]: done

File: Sources/Views/ALKFriendFormCell.swift (hunk 2 above)
Comment (Sathyan-Elangovan) [reply Sources/Views/ALKFriendFormCell.swift]: done

**Review by mukeshthawani — APPROVED** [review]

**Pull request merged**